index.js

```diff
@@ -7,14 +7,20 @@ import { convertToMs, calculateDelay } from './lib/helpers.js'
 
 const slowDownPlugin = async (fastify, settings) => {
   const options = { ...DEFAULT_OPTIONS, ...settings }
-  const store = new Store(convertToMs(options.timeWindow))
+  const store = new Store(
+    convertToMs(options.timeWindow),
+    convertToMs(options.intevalTimeExpiredKeys),
```

Comment:
Maybe should be `intervalTimeExpiredKeys`, I believe there is a typo here.

Reply:
Yep ;) Good catch !